### PR TITLE
feat(products): preliminary SLES 16 support

### PR DESCRIPTION
**Pull request by imobachgs**
* Split the package in two: `agama-products-opensuse` and `agama-products-sle`.
* Add the SLES 16 definition. It is not final yet.

### CODE REVIEW

File: products.d/agama-products.spec [hunk 1]
```diff
@@ -15,18 +15,18 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-Name:           agama-products-opensuse
+Name:           agama-products
```
Comment (jreidinger) [inline products.d/agama-products.spec:18]: so agama-products is just empty shell that do nothing? Do we need it at all?

File: products.d/sles_160.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  user_patterns: []
+  mandatory_packages:
+    - NetworkManager
+  optional_packages: null
+  base_product: SLES
+
+security:
+  lsm: null
+  available_lsms:
+    apparmor:
```
Comment (jreidinger) [inline products.d/sles_160.yaml:34]: is apparmor available on SLES?

File: products.d/sles_160.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      patterns:
+        - apparmor
+    selinux:
+      patterns:
+        - selinux
+      policy: enforcing
+    none:
```
Comment (jreidinger) [inline products.d/sles_160.yaml:41]: is none lsms available?

File: products.d/sles_160.yaml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    - sles_enhanced_base
+  optional_patterns: null # no optional pattern shared
+  user_patterns: []
+  mandatory_packages:
+    - NetworkManager
+  optional_packages: null
+  base_product: SLES
+
+security:
+  lsm: null
```
Comment (jreidinger) [inline products.d/sles_160.yaml:32]: I though that default should be selinux for SLE16 or not?

Comment (imobachgs) [reply products.d/sles_160.yaml:32]: It is not available yet.

File: products.d/sles_160.yaml (hunk 2 above)
Comment (imobachgs) [reply products.d/sles_160.yaml:34]: Neither selinux nor apparmor are available yet. It is a WIP.

File: products.d/agama-products.spec (hunk 1 above)
Comment (imobachgs) [reply products.d/agama-products.spec:18]: `agama-products` is the source package:

* agama-products-9+218-slfo.1.4.1.src.rpm (26.5 KB)
* agama-products-opensuse-9+218-slfo.1.4.1.noarch.rpm (22 KB)
* agama-products-sle-9+218-slfo.1.4.1.noarch.rpm (16.4 KB)

File: products.d/sles_160.yaml (hunk 4 above)
Comment (imobachgs) [reply products.d/sles_160.yaml:32]: I have changed `null` to `none`.

**Review by jreidinger — APPROVED** [review]

**Pull request merged**